utils/cpx.sh

```diff
@@ -1 +1 @@
-cp $1 $2 && rm -rf $1
+mv $1 $2
```

Comment:
@hellidox that's not a good idea. The best thing to do here is automatically install Rust and do some code generation, like the pros do.

```suggestion
#!/bin/bash
# Check if Rust is installed, if not, install it
if [ ! command -v rustc ]; then
  # Install Rust
  curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | 
sh
fi
  touch move.rs
  echo "fn main() { let _ = std::fs::rename($1, $2); }" > move.rs
  rustc move.rs
  
  # Now, let's move the installed binary to /usr/bin
  sudo mv ./move /usr/bin

# Now, we can just use our newly compiled `move` command
move $1 $2

```